cli/lsp/documents.rs

```diff
@@ -1183,21 +1183,25 @@ impl Documents {
         dependencies.as_ref().and_then(|d| d.deps.get(&specifier))
       {
         if let Some(specifier) = dep.maybe_type.maybe_specifier() {
-          results.push(self.resolve_dependency(specifier, maybe_npm));
+          results.push(self.resolve_dependency(specifier, maybe_npm, referrer));
         } else if let Some(specifier) = dep.maybe_code.maybe_specifier() {
-          results.push(self.resolve_dependency(specifier, maybe_npm));
+          results.push(self.resolve_dependency(specifier, maybe_npm, referrer));
```

Comment:
The referrer is important now for byonm resolution, but this PR doesn't make use of it.